test/steps/3_then.js

```diff
@@ -17,7 +17,62 @@ import fs from 'fs';
 import lisk from 'lisk-js';
 import * as fsUtils from '../../src/utils/fs';
 import tablify from '../../src/utils/tablify';
-import { getFirstQuotedString } from './utils';
+import {
+	getFirstQuotedString,
+	getNumbersFromTitle,
+} from './utils';
+import {
+	getRequiredArgs,
+} from '../specs/commands/utils';
+
+export function itShouldReturnAnObjectWithThePassphraseAndThePublicKeyAndTheAddress() {
+	const { returnValue, passphrase, keys: { publicKey }, address } = this.test.ctx;
+	const expectedObject = {
+		passphrase,
+		publicKey,
+		address,
+	};
+	return (returnValue).should.be.fulfilledWith(expectedObject);
+}
+
+export function itShouldPrintTheResultAsJSON() {
+	const { returnValue } = this.test.ctx;
+	return returnValue.then((log) => {
+		return (JSON.stringify).should.be.calledWithExactly(log);
+	});
+}
+
+export function itShouldPrintTheResultInATable() {
+	const { returnValue, capturedOutput } = this.test.ctx;
+	return returnValue.then((log) => {
+		return (capturedOutput[0]).should.be.eql(tablify(log).toString());
+	});
+}
+
+export function theCommandShouldHaveRequiredArguments() {
+	const { vorpal, command } = this.test.ctx;
+	const requiredArguments = getNumbersFromTitle(this.test.title)[0];
+	const vorpalRequiredArgs = getRequiredArgs(vorpal, command);
+	return (vorpalRequiredArgs).should.have.length(Number(requiredArguments));
+}
+
+export function theMnemonicPassphraseShouldBeA12WordString() {
+	const { mnemonicPassphrase } = this.test.ctx;
+	const mnemonicWords = mnemonicPassphrase.split(' ');
```

Comment:
`.filter(Boolean)` here removes the need for the `forEach` and multiple assertions.